css/layout.css

```diff
@@ -13,12 +13,24 @@
   flex-wrap: wrap;
 }
 
+.column-full {
+  flex-basis: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.column-half {
+  flex-basis: 50%;
+  display: flex;
+  flex-direction: column;
+}
+
```

Comment:
We already provided you `column-full` and `column-half` classes to use.

If you want a flex column, this should be a separate utility class then. You could call this `column` or `flex-column` and give it `display: flex` and `flex-direction: column` but your utility classes should do the one job they are meant to do.